### PR TITLE
junk food affects score negatively

### PR DESCRIPTION
**Pull request by karinamzalez**
- add score attribute to junk food
- add junkCollisionOutcome
- add isJunkFoodCollidingWithTray
- add decreaseScore

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -125,27 +123,56 @@ function junkFoodMove(junkFood){
...
-      tray.y > healthFood.y + healthFood.height ||
-      tray.y + tray.height < healthFood.y);
-}
+window.setInterval(collisionOutcome, 200, healthFoods, world.tray);
 
 function collisionOutcome(healthFoods, tray){
-  for (var i = 0; i < healthFoods.length; i++)
-  {var healthFood = healthFoods[i];
+  for (var i = 0; i < healthFoods.length; i++){
+    var healthFood = healthFoods[i];
```
Comment (chompasina) [inline lib/index.js:130]: I wonder if this is something we could do with junkFood in the same function too?

File: lib/index.js [hunk 2]
```diff
@@ -125,27 +123,56 @@ function junkFoodMove(junkFood){
...
+  for (var i = 0; i < junkFoods.length; i++){
+    var junkFood = junkFoods[i];
+    if (isJunkFoodCollidingWithTray(junkFood, tray)){
+      decreaseScore(junkFood);
+    }
+  }
+}
+
+function isJunkFoodCollidingWithTray(junkFood, tray){
+  return !(
```
Comment (chompasina) [inline lib/index.js:165]: I think we could refactor this with the healthFood one again to save ourselves from repetitive code. Looks great!

**Comment by chompasina** [comment]
Let's merge! This adds functionality for junk food can delete from the score. We can refactor our functions into the healthFood functions in the next commit.

File: lib/random-junk-food.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
 function RandomJunkFood(x){
   var junkFoods = [
-    {x: x, "type": "burger", "color": "saddlebrown"},
-    {x: x, "type": "hotdog", "color": "tan"},
-    {x: x, "type": "cake", "color": "black"},
-    {x: x, "type": "friedChicken", "color": "dimgray"},
-    {x: x, "type": "pizza", "color": "salmon"}
+    {x: x, "type": "burger", "color": "saddlebrown", "score": "5"},
```
Comment (deborahleehamel) [inline lib/random-junk-food.js:3]: I like the idea of adding a score to the junk foods also. Let's look up and have another relative scale for different negative amounts for each junk food.

Comment (chompasina) [reply lib/random-junk-food.js:3]: Great thinking @deborahleehamel !

**Comment by deborahleehamel** [comment]
Yes, let's merge and keep moving forward with the refactor for health foods. Thanks!

**Pull request merged**